doc/source/api/core/index.rst

```diff
@@ -2,6 +2,12 @@
 
 Core
 ====
+
+This section describes the core pythonic interfaces for Fluent. 
```

Comment:
```suggestion
This section describes the core Pythonic interfaces for Fluent. 
```

Reply:
@smf2001 @dnwillia-work I think we still need to address terminology. I believe there is one API for Fluent-Core. It consists of multiple resources (methods, functions). If this is accurate, the above sentence should be something like "This section describes the Fluent API (application programming interface).

Reply:
Yes, the ansys-fluent-core package provides what we call the 'core' APIs.

I would probably write "This section describes the Fluent Python API (application programming interface)" since we also have the gRPC API under the hood.